src/cordova/apple/xcode/osx/Outline.xcodeproj/project.pbxproj

```diff
@@ -757,7 +757,7 @@
 					/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include,
 					"\"$(BUILD_ROOT)/../IntermediateBuildFilesPath/UninstalledProducts/include\"",
 				);
-				MACOSX_DEPLOYMENT_TARGET = 10.11;
+				MACOSX_DEPLOYMENT_TARGET = 10.14;
```

Comment:
does this affect our supported ios version? safari tends to be updated in lockstep with the os version

Reply:
It only affects macOS. iOS is 9.0.
I don't see any data for macOS <10.14 on the app Store, and it's old.